pallets/asset-manager/src/lib.rs

```diff
@@ -52,7 +52,7 @@ pub mod pallet {
     use frame_system::pallet_prelude::*;
```

Comment:
Up to you, but it would make reviewing much simpler if we can split this out to:
1. asset-manager changes PR
2. manta-support changes PR
3. add manta-sbt PR

Reply:
Now is only

2. manta-support changes PR
3. add manta-sbt PR

Reply:
I forget what the asset-manager changes were exactly, but do we need to log them separately ?

Reply:
I reverted this change and moved the logic to `pallet-manta-sbt` as it is a different ledger and now have overlapping `AssetIds` with `MantaPay`

The `asset-manager` change was mainly adding an `AssetMetadata`  enum to allow for different types of metadata in storage. This change maybe still useful in the future.